src/modules/cpu/host_tensor_augmentations.hpp

```diff
@@ -9261,16 +9261,16 @@ RppStatus spatter_u8_u8_host_tensor(Rpp8u *srcPtr,
 
         std::random_device rd;  // Random number engine seed
         std::mt19937 gen(rd()); // Seeding rd() to fast mersenne twister engine
-        std::uniform_int_distribution<> distribX(0, 1920 - roiPtr->xywhROI.roiWidth);
-        std::uniform_int_distribution<> distribY(0, 1080 - roiPtr->xywhROI.roiHeight);
+        std::uniform_int_distribution<> distribX(0, SPATTER_MAX_WIDTH - roiPtr->xywhROI.roiWidth);
```

Comment:
Is SPATTER_MAX_WIDTH and height user parameters?

Reply:
No, the SPATTER_MAX_WIDTH is defined at compile time. 1920 for now.